### PR TITLE
Improved README.md

### PR DESCRIPTION
**Pull request by Jprillaman**
#### Brief description of what is fixed or changed
Replaced Who are we? and How are we different? with an About section with: "Currently, the aim of the project is to be a Python package for various data structures in computer science. In addition, we are also working on including parallel algorithms. To the best of our knowledge there doesn't exist any well designed library/package which has covered most of the data structures and algorithms including their parallel implementation. 

In future, i.e, after a few releases of the package when the software design will become stable, we also aim to provide APIs for the code in C++ and Java as well. For clarity, the interested students aren't expected to know C++ and Java. Knowledge of Python is enough for this project."

### CODE REVIEW

**Comment by czgdp1807** [comment]
@Jprillaman Thanks for your contributions. Are you participating through GSSoC 2020?

**Comment by Jprillaman** [comment]
No I'm not. I hope that isn't a problem. I've just started learning how to use Github

**Comment by czgdp1807** [comment]
@Jprillaman No problems at all. Actually, we have a separate label for their participants to track their progress. Thanks for the contribution. :-)

File: README.md [hunk 1]
```diff
@@ -3,15 +3,12 @@ PyDataStructs
...
+Currently, the aim of the project is to be a Python package for various data structures in computer science. In addition, we are also working on including parallel algorithms. To the best of our knowledge there doesn't exist any well designed library/package which has covered most of the data structures and algorithms including their parallel implementation. 
 
-How are we different?
----------------------
-
-There are many pre-exisiting packages available in the open source world based on the above idea. However, they lack the implementation of complex data structures and this makes us different. If you have worked with C++ and Python then you know how hard it is to code bug free AVL trees :-).Well, after this project you will not have to worry about it. In fact, we will keep each data structure independent from each other for easy code reusability.
+In future, i.e, after a few releases of the package when the software design will become stable, we also aim to provide APIs for the code in C++ and Java as well. For clarity, the interested students aren't expected to know C++ and Java. Knowledge of Python is enough for this project.
```
Comment (czgdp1807) [inline README.md]: Please remove "For clarity, the interested students aren't expected to know C++ and Java. Knowledge of Python is enough for this project."

Comment (Jprillaman) [reply README.md]: @czgdp1807 I've removed what you asked. Not sure if you can see the change

Comment (Jprillaman) [reply README.md]: @czgdp1807 Do I need to start a new pull request or was the file change enough?

**Comment by czgdp1807** [comment]
Thanks. Merging.

**Pull request merged**